tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -913,3 +917,74 @@ func createStaticPvAndPvcInParallel(client clientset.Interface, ctx context.Cont
 
 	}
 }
+
+// triggerFullSyncInParallel triggers full sync on demand in parallel
+func triggerFullSyncInParallel(ctx context.Context, client clientset.Interface, wg *sync.WaitGroup) {
+	defer wg.Done()
+	restConfig := getRestConfigClient()
+	crd := getTriggerFullSyncCrd(ctx, client, restConfig)
+	updateTriggerFullSyncCrd(ctx, restConfig, crd)
+	err := waitForFullSyncToFinish(client, ctx, restConfig)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "Full sync did not finish in given time")
+	crd = getTriggerFullSyncCrd(ctx, client, restConfig)
+	updateTriggerFullSyncCrd(ctx, restConfig, crd)
+}
+
+// getTriggerFullSyncCrd fetches full sync crd from the list of crds in k8s cluster
+func getTriggerFullSyncCrd(ctx context.Context, client clientset.Interface,
+	restConfig *rest.Config) triggercsifullsyncv1alpha1.TriggerCsiFullSync {
+	cnsOperatorClient, err := k8s.NewClientForGroup(ctx, restConfig, cnsoperatorv1alpha1.GroupName)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	crds := &triggercsifullsyncv1alpha1.TriggerCsiFullSyncList{}
+	err = cnsOperatorClient.List(ctx, crds)
+	framework.Logf("Error is %v, list of crds is %v", err, crds)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	crdFound := false
+
+	var fullSyncCrd triggercsifullsyncv1alpha1.TriggerCsiFullSync
+	for _, crd := range crds.Items {
+		framework.Logf("crd name is %v", crd.Name)
+		if crd.Name == crdtriggercsifullsyncsName {
+			framework.Logf("Found TriggerFullSync crd: %v", crd.Name)
+			fullSyncCrd = crd
+			crdFound = true
+			break
+		}
+
+	}
+	gomega.Expect(crdFound).To(gomega.BeTrue(), "couldn't find full sync crd: %s", crdtriggercsifullsyncsName)
```

Comment:
once you change to use `Get()` instead of `List()` above, change
```golang
gomega.Expect(crdFound).To(gomega.BeTrue(), "couldn't find full sync crd: %s", crdtriggercsifullsyncsName)
```
to
```golang
gomega.Expect(crd).NotTo(gomega.BeNil(), "couldn't find full sync crd: %s", crdtriggercsifullsyncsName)
```